app/models/user.rb

```diff
@@ -39,6 +39,11 @@ def github_client_scopes
     GitHubUser.new(github_client, uid).client_scopes
   end
 
+  # for searching, disable pagination so we can control the number of suggestions
+  def github_search_client
```

Comment:
Yeah, I see that we need the access token from the user, but why does this client belong on the user? Couldn't you also do something in the controller like this

```ruby
def search_client
  Octokit::Client.new(access_token: current_user.token, auto_paginate: false)
end
```

I'm not sure why, but I don't really like adding this method here to use it in just one case. Alternatively, what about adding an options hash argument to `User#github_client` that would allow you to disable pagination with that method?